### ISSUE TITLE
Disabling unknown field warnings (`TIFFSetWarningHandler` option)

### ISSUE THREAD

**Issue by Padoup-Padoup** [pearu/pylibtiff#50]
Hi,

When reading an image stack with `libtiff.TIFF` I often get the following warning message: 

```
TIFFReadDirectory: Warning, Unknown field with tag 32781 (0x800d) encountered.
TIFFFetchNormalTag: Warning, ASCII value for tag "ImageDescription" does not end in null byte.
```

I have read ([here](http://public.kitware.com/pipermail/insight-users/2007-August/023376.html)) that the original `libtiff` library provides a way to disable them through the option `TIFFSetWarningHandler(0)`, however, I cannot find the option in pylibtiff. Is this option available?

If not, this option would really be helpful: when loading a movie of 3D pictures, it frequently happens that I get thousands of warning that overflow my console.
Thank you

**Comment by cmbruns-hhmi** [comment]
```
import libtiff
libtiff.libtiff_ctypes.suppress_warnings()
```

Does the trick for me.

**Comment by Padoup-Padoup** [comment]
Many thanks. Do you know where it is documented (I searched quite a bit without finding). I will try and let you know how it goes.

**Comment by cmbruns-hhmi** [comment]
@Padoup-Padoup I did not find it documented anywhere. 

Just today I found this as I was in the process of trying to excavate the naked ctypes libtiff handle, so I could directly call libtiff.TIFFSetWarningHandler(0) on it. But on the way I found this method created back in 2010:

https://github.com/pearu/pylibtiff/blob/master/libtiff/libtiff_ctypes.py#L1522

I recommend that this `suppress_warnings()` method be elevated up to the top level libtiff scope.
